ddl/ddl_db_test.go

```diff
@@ -1376,6 +1376,45 @@ func (s *testDBSuite) TestTruncateTable(c *C) {
 	c.Assert(hasOldTableData, IsFalse)
 }
 
+func (s *testDBSuite) TestAlterTableOption(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.mustExec(c, "use test")
+	s.mustExec(c, "create table alter_table (c1 int, c2 int)")
+
+	ctx := s.tk.Se.(context.Context)
+	is := domain.GetDomain(ctx).InfoSchema()
```

Comment:
Now why you can pass tests is because you add `handleTableOptions` in the function of `AlterTableOption`.
You'd better put it after line 1391.

Reply:
I removed `handleTableOptions` from `AlterTableOption`, but test failed.

It seems that new schema can NOT be reloaded after DDL changes.